lib/new_relic/telemetry_sdk/harvester.rb

```diff
@@ -23,6 +25,9 @@ def register name, buffer, client
             client: client
           }
         end
+      rescue => e
+        logger.error "Encountered error while registering buffer #{name}."
+        logger.error e
```

Comment:
Missed one!  :-)

Reply:
Great eye! Thanks for noticing that. Fixed!